docs/intro.rst

```diff
@@ -190,6 +198,11 @@ Requirements
 Contribute
 ----------
 
+.. warning::
+
+   Because Camus is deprecated both in Confluent Platform and upstream, only critical bug fixes are accepted to these
```

Comment:
Yeah, but we know they won't accept patches anymore (really at all, although I think for something really bad we could get someone to listen). I just wanted to emphasize that all avenues for updates are basically closed at this point.